draft-packages/modal/KaizenDraft/Modal/Primitives/GenericModal.tsx

```diff
@@ -150,4 +157,70 @@ class GenericModal extends React.Component<Props> {
   }
 }
 
+/**
+ * Get an element's owner document. Useful when components are used in iframes
+ * or other environments like dev tools.
+ *
+ * @param element
+ */
+function getOwnerDocument<T extends HTMLElement = HTMLElement>(
+  element: T | null
+) {
+  return element && element.ownerDocument
+    ? element.ownerDocument
+    : canUseDOM()
+    ? document
+    : null
+}
+
+function canUseDOM(): boolean {
+  return (
+    typeof window !== "undefined" &&
+    typeof window.document !== "undefined" &&
+    typeof window.document.createElement !== "undefined"
+  )
+}
+
+// tslint:disable-next-line: no-empty
+function noop(): void {}
+
+function createAriaHider(dialogNode: HTMLElement) {
```

Comment:
Would be quite helpful to have a comment at the top of this to briefly explaining what this does and why (same with the `canUseDom` function above).